src/index.json

```diff
@@ -1372,8 +1372,8 @@
         ],
         "dms-preview": [
```

Comment:
As you know, dms-preview value is an array, I will prefer append your new download link with previous one. In this way. we can keep the capability to install with previous version.

Reply:
We don't really want to retain the previous version since it would enable support for a deprecated scenario. This change is meant to replace the default request for the PG scenario to a different call.